pkg/container/bytejson/bytejson.go

```diff
@@ -26,14 +26,17 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/common/moerr"
 )
 
-func (bj ByteJson) String() string {
-	ret, _ := bj.MarshalJSON()
-	return string(ret)
+func (bj ByteJson) String() (string, error) {
```

Comment:
Why String return (string, error)?

Reply:
because in tae.go/ReadLine(),Line351:
If json is invalid, it will stop read and return immediately.
but Bytejson.String() doesn't return error now, it will make unstoppable error log.